src/main/java/frc/robot/systems/BallHandlingFSM.java

```diff
@@ -79,10 +84,10 @@ public BallHandlingFSM() {
 		pDH = new PowerDistribution(1, ModuleType.kRev);
 
 		// Reset state machine
-		pushCommandTimeStamp = Timer.getFPGATimestamp() - Constants.PUSH_TIME_SECONDS;
+		pushCommandTimeStamp = Timer.getFPGATimestamp() - Constants.TIME_FOR_FULL_SHOT;
```

Comment:
I think this would be more readable if a separate Timer object was instantiated and started/reset at the correct transitions. But in principle I guess this works

Reply:
Could we fix this in next PR, I noticed this as well and was hoping we could update it at a later date